torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1083,16 +1064,16 @@ def clip_grad_norm_(
         Returns:
             Total norm of the parameters (viewed as a single vector).
 
-        .. note:: If every FSDP instance uses ``NO_SHARD``, meaning that no
+            If every FSDP instance uses ``NO_SHARD``, meaning that no
```

Comment:
Would this get rendered under `Returns`?

Reply:
Oops, fixed.